core/ir/aarch64/instr_create_api.h

```diff
@@ -1693,6 +1693,56 @@ enum {
 #define INSTR_CREATE_bif_vector(dc, Rd, Rm, Rn) \
     instr_create_1dst_2src(dc, OP_bif, Rd, Rm, Rn)
 
+/**
+ * Creates a FCVTAS vector instruction.
+ * \param dc      The void * dcontext used to allocate memory for the instr_t.
```

Comment:
Use `#` to make a Doxygen link: `#instr_t`.
Ditto below.

(I think some existing code is missing this: I filed #4856.)

I assume these will be auto-generated long-term.

Reply:
Done